src/lib/storage/value_segment.cpp

```diff
@@ -13,30 +13,36 @@
 
 namespace opossum {
 
+
+
 template <typename T>
 AllTypeVariant ValueSegment<T>::operator[](const ChunkOffset chunk_offset) const {
   PerformanceWarning("operator[] used");
-  return value_segment.at(chunk_offset);
+  return _value_segment.at(chunk_offset);
```

Comment:
One would argue, that `operator[]` implies the underlying function also uses `operator[]`